testapps/SimpleApp/SimpleApp.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
```

Comment:
It looks like every testapp has been updated to multi-target. Do you have any test apps that are only single TFMS? Would be good to keep some of these test cases around as that is the more common scenario.